gui/src/components/CreateBackup.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+    <div class="modal fade" tabindex="-1" id="registerBackup" aria-labelledby="registerBackupModal" aria-hidden="true">
+        <div class="modal-dialog modal-dialog-centered">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h5 class="modal-title">Start a new backup</h5>
+                    <button class="close" type="button" data-dismiss="modal" aria-label="Close">
+                        <span aria-hidden="true">&times;</span>
+                    </button>
+                </div>
+                <div class="modal-body">
+                    <form>
+                        <div class="form-group">
+                            <label for="backupTypeSelect">Select a backup type</label>
+                            <select class="custom-select" id="backupTypeSelect" v-model="backup_type">
+                                <option selected :value="2">Snapshot</option>
+                                <option :value="1">Cluster-Wide</option>
```

Comment:
```suggestion
                                <option :value="1">Cluster-wide</option>
```
for consistency with [here](https://github.com/scalar-labs/cassy/pull/46/files#diff-1d6c093de68c381c8d652e660be617eaR101).